doc/task_breakdown.md

```diff
@@ -0,0 +1,70 @@
+# Task Breakdown for SQL Query Builder Development
+
+## 1. Core Modules of the Program
+
+### 1.1 Table Management Module
+- Tasks:
+  - Implement functionality to display a list of tables from the database.
+  - Add the ability to select multiple tables simultaneously.
+  - Configure the removal of tables from the current selection set.
+- Technical Details:
```

Comment:
Why I still see this section? 
each task should be atomic, and you should be able to perform it without references to something except goals, requirements or design considerations (that are also requirements)